app/api/orders.py

```diff
@@ -0,0 +1,228 @@
+from datetime import datetime
+
+from flask import request
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from marshmallow_jsonapi.flask import Schema, Relationship
+from marshmallow_jsonapi import fields
+from marshmallow import post_dump, validates_schema, validate
+from flask_jwt import current_identity as current_user
+
+from app.api.bootstrap import api
+from app.api.data_layers.ChargesLayer import ChargesLayer
+from app.api.helpers.db import save_to_db, safe_query
+from app.api.helpers.exceptions import ForbiddenException, UnprocessableEntity
+from app.api.helpers.payment import PayPalPaymentsManager
+from app.api.helpers.ticketing import TicketingManager
+from app.api.helpers.permission_manager import has_access
+from app.api.helpers.permissions import jwt_required
+from app.api.helpers.utilities import dasherize, require_relationship
+from app.models import db
+from app.models.discount_code import DiscountCode, TICKET
+from app.models.order import Order, OrderTicket
+
+
+class OrderSchema(Schema):
+    class Meta:
+        type_ = 'order'
+        self_view = 'v1.order_detail'
+        self_view_kwargs = {'id': '<id>'}
+        inflect = dasherize
+
+    @post_dump
+    def generate_payment_url(self, data):
+        if 'POST' in request.method or ('GET' in request.method and 'regenerate' in request.args) and 'completed' != \
+           data["status"]:
+            if data['payment_mode'] == 'stripe':
+                data['payment_url'] = 'stripe'
```

Comment:
Mmm ... Let this follow a specific URI Scheme ... Would be easier for clients, if there are parsers on the client side that look for URIs

Make this `stripe://payment` (similar to an application URI scheme present in android and windows for example)